### ISSUE TITLE
Lucid displays poorly on small screens

### ISSUE THREAD

**Issue by ZLLentz** [pcdshub/lucid#57]
If the grid is larger than the screen height, you can't navigate to the Quick Launcher Toolbar on some setups. This can also be a problem for the device windows that open up to the right if the screen is not wide enough.

**Comment by klauer** [comment]
This is really painful and bad on FastX:

<img width="886" alt="image" src="https://user-images.githubusercontent.com/5139267/83814682-976a5700-a673-11ea-934c-97bf75424c3c.png">

Can't resize the quick launcher toolbar to even see the full grid.

**Comment by hhslepicka** [comment]
That is odd... Yesterday night I was able to pull the Quick Launcher toolbar to be small and see the fullgrid... clearly it is a bug that it is showing this big but the resizing was working properly.
I will take a look into that.

**Comment by klauer** [comment]
Looking back at this, it's very possible that what I saw above was related to #67 and just didn't allow for any further mouse input.

**Comment by klauer** [comment]
Not resolved. Strange overlap of grid + menu options:

<img width="1468" alt="image" src="https://user-images.githubusercontent.com/5139267/91219789-11890580-e6d0-11ea-98b0-6b61b111e3dd.png">

**Comment by hhslepicka** [comment]
Oh damn! In which case did this happen? Given that you have a screenshot, can you send some instructions to reproduce that? This would make it easier to debug and find a solution.

Just a quick pool here: @klauer @ZLLentz 
If the screen is so small that it can't fit info, should we add scrollbars to the grid?

**Comment by klauer** [comment]
Reproduce by running `lfe` on a 15" macbook pro (or just set your screen resolution lower if you can't reproduce).

I think the grid shouldn't be so massive if it only has sparse dots as does the example, should prefer showing its full contents but fallback to scrollbars if necessary.

**Comment by hhslepicka** [comment]
I will work on that.

**Comment by ZLLentz** [comment]
I kind of want to edit happi and group things more aggressively in that grid...

**Comment by klauer** [comment]
It goes without saying users should still be able to access the functionality regardless of their screen resolution.

But - it's @ZLLentz's call here for the exact direction this takes.

**Comment by hhslepicka** [comment]
I agree with you, @klauer. Users should ultimately be able to operate the screens no matter which resolution they have.

**Comment by hhslepicka** [comment]
I managed to reproduce it here following your guidance, @klauer and it looks awful (as expected).
Working on a fix right now.
<img width="1118" alt="Screen Shot 2020-08-25 at 2 55 45 PM" src="https://user-images.githubusercontent.com/8185425/91232072-3fc41080-e6e3-11ea-9aa3-ea8bf9db0289.png">
